samples/nrf9160/http_application_update/src/main.c

```diff
@@ -97,10 +117,78 @@ static void app_dfu_transfer_start(struct k_work *unused)
 					     DT_GPIO_PIN(DT_ALIAS(sw0), gpios),
 					     GPIO_INT_EDGE_TO_ACTIVE);
 
+		printk("fota_download_start() failed, err %d\n", retval);
+		return;
+	}
+
+}
+
+static void apply_fmfu(void)
+{
+	int err;
+
+	err = fmfu_fdev_load(0);
+	if (err != 0) {
+		printk("fmfu_fdev_load failed: %d\n", err);
+		return;
+	}
+
+	printk("Modem firmware update completed, please restart the device");
```

Comment:
This is the way it works for MCUBoot updates. 

IMO this is better, because otherwise you need to scroll back past the rather long SPM log printed during boot to see what actually happened.